spec/lib/jobs/install-os-job-spec.js

```diff
@@ -379,5 +379,28 @@ describe('Install OS Job', function () {
                    .to.throw('The uid should between 500 and 65535 (>=500, <=65535)');
         });
         
+        it('should throw error when mountPoint is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:{}, size:'500', fsType:'ext3' }];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('mountPoint must be a string');
+        });
+
+        it('should throw error when size is not a string', function() {
+            job.options.installPartitions = [{ mountPoint:'/boot', size:{}, fsType:'ext3' }];
```

Comment:
Sorry, I disagree here. I think one of the key point for unittest case is  'unit'. and more for what' goot unit test case, what' bad unit test case, one reference is cycle complexity.